thumbnails.yaml

```diff
@@ -0,0 +1,68 @@
+packages:
```

Comment:
is there a reason thumbnails should be in a separate remake file instead of being part of viz.yaml? I would think they would follow the `process`, `visualize`, `publish` chain

Reply:
Since I was using the code from the national pie map figure generation which is a separate remake, I just kept the same pattern. I can't think of why that would be a problem, so I can do that.

Reply:
well actually, didn't we keep that original map one separate since it's using different spatial data. If that isn't a reason to keep them separate then what is keeping us from adding both the national map figure generation and gif generation to the regular viz yaml? I'm kind of leaning towards not now that I remembered that spatial thing.